internal/installcfg/builder/builder_test.go

```diff
@@ -609,10 +609,12 @@ func getInventoryStr(hostname, bootMode string, ipv4 bool, ipv6 bool) string {
 		},
 	}
 	if ipv4 {
-		inventory.Interfaces[0].IPV4Addresses = []string{"10.35.20.10/24"}
```

Comment:
All the tests are using `1.2.3.0/24` as machine network, so we need to have a hosts' interface in this subnet